src/content/en/2020/http2.md

```diff
@@ -4,20 +4,534 @@ part_number: IV
+
+However, even if it is known that a server supports QUIC and HTTP/3, the network in-between might block it. UDP traffic is commonly used in DDoS attacks and blocked by default in for example many company networks. While exceptions could be made for QUIC, its encryption makes it difficult for firewalls to assess the traffic. There are potential solutions to these issues but in the meantime it is expected that QUIC is most likely to succeed on well-known ports like 443. And it is entirely possible that it is blocked QUIC altogether. In practice, clients will likely use sophisticated mechanisms to fall back to TCP if QUIC fails. One option there is to “race” both a TCP and QUIC connection and use the one that completes first.
+
+There is ongoing work to define ways to discover HTTP/3 without needing the TCP step. This should be considered an optimization though, as the UDP blocking issues are likely to mean that TCP-based HTTP sticks around. The [HTTPS DNS record](https://tools.ietf.org/html/draft-ietf-dnsop-svcb-https), is similar to HTTP Alternative Services and some CDNs are already [experimenting with these records](https://blog.cloudflare.com/speeding-up-https-and-http-3-negotiation-with-dns/).  In the long run, when most servers offer HTTP/3, browsers might switch to attempting that by default. But that will take a long time.
+
+QUIC is dependent on TLS1.3, which is used for around 41% of requests, which means the other 60% of requests will need to update their TLS stack to support HTTP/3.
+
+#### Dual stack deployment implications
+
+From the discussion above, we expect QUIC and TCP to continue running in parallel for some time. We expect servers (e.g., nginx/apache/nodejs) to offer easy ways to do so, but this is still likely to complicate localhost test setups. Furthermore, there's probably a longer term impact on the HTTP Archive and WebPageTest projects.  The amount of pages crawled will increase substantially if we want to crawl both the HTTP/3 and H2/H1 versions, or the runtimes will increase if we want to prioritize HTTP/3.
+
+Currently, some browsers provide (temporary) command line arguments to force QUIC on a certain origin to make this easier, but this is not the most user-friendly option of course. Over time, this will probably become easier to configure. For example, this command will force Chrome to try a QUIC connection to localhost immediately without having to discover it via alternative services :
+
+```
+chrome --enable-quic --quic-version=h3-29 --origin-to-force-quic-on=localhost:6121 http://localhost:6121/
```

Comment:
Does the target need to be `https`? Maybe @rmarx knows.

Reply:
Good point, and I assume it does need to be https. Even if it isn't a hard requirement, it makes more sense to make it https:// here